### PR TITLE
Day35 activities

### PR DESCRIPTION
**Pull request by sharonjohnson007**
please review sir

### CODE REVIEW

File: Day35/src/main/java/Search.java [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+    public static boolean binarySearch(int[] array, int key) {
+        for (int i = 0; i < array.length; i++) {
+            if (array[i] == key) {
+                return true;
+            }
+        }
+        return false;
+    }
+}
```
Comment (PushpaNikki) [inline Day35/src/main/java/Search.java:62]: This is not the expected implementation

Comment (dorarajgcit) [inline Day35/src/main/java/Search.java:59]: This is not the expected implementation

File: Day35/src/main/java/Sorting.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            }
+            int temp = array[min];
+            array[min] = array[i];
+            array[i] = temp;
+        }
+        for (int i = 0; i < array.length; i++) {
+            System.out.print(array[i] + " ");
+        }
+        System.out.println();
+    }
```
Comment (dorarajgcit) [inline Day35/src/main/java/Sorting.java:57]: Come up with a different approach

**Review by dorarajgcit — COMMENTED** [review]
Added comments, please address them

**Pull request merged**